database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Models/PostViewModel.swift

```diff
@@ -62,6 +64,27 @@ class PostViewModel: ObservableObject, Identifiable {
     self.userIDsStarredBy = userIDsStarredBy
   }
 
+  func didTapSendButton(commentField: String) {
+    if let userID = Auth.auth().currentUser?.uid,
+      let userEmail = Auth.auth().currentUser?.email {
+      commentRef = Database.database().reference().child("post-comments").child(id)
+      guard let key = commentRef.childByAutoId().key else { return }
+      let comment = ["uid": userID,
+                     "author": userEmail,
+                     "text": commentField]
+      commentRef.child(key).setValue(comment)
+    }
+  }
+
+  func fetchComments() {
+    commentRef = Database.database().reference().child("post-comments").child(id)
+    refHandle = commentRef.observe(DataEventType.value, with: { snapshot in
+      guard let comments = snapshot.value as? [String: [String: Any]] else { return }
+      let sortedComments = comments.sorted(by: { $0.key > $1.key })
```

Comment:
Is the key ordered in time?

Reply:
Yes. Similar to the posts comments are ordered from most recent to least recent.